community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/NativeLabelScanStore.java

```diff
@@ -426,6 +443,12 @@ public void stop() throws IOException
     @Override
     public void shutdown() throws IOException
     {
+        boolean flush = started && health.isHealthy();
+        monitor.flushDuringShutdown( flush );
```

Comment:
why do we call this thing flush instead of checkpoint? wouldn't that be a bit confusing?